cmd2/parsing.py

```diff
@@ -0,0 +1,387 @@
+#
+# -*- coding: utf-8 -*-
+"""Statement parsing classes for cmd2"""
+
+import re
+import shlex
+from typing import List, Tuple
+
+import cmd2
```

Comment:
Strongly recommend removing import of ``cmd2``.  First off, it isn't used in this file.  Secondly, it results effectively in a circular import loop.

Reply:
Import removed and pushed to `ply` branch.